lib/code_corps/adapter/map_transformer.ex

```diff
@@ -0,0 +1,86 @@
+defmodule CodeCorps.Adapter.MapTransformer do
```

Comment:
We were using this module as a Stripe util module, but the approach it uses is generic and can be reused for the purposes of GitHub adapters, so I moved it to outside of Stripe's scope.

I also did some renaming and added documentation to make it clearer what the functions in it actually do.